alpine/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine
+RUN apk add --update curl python bash &&  rm -rf /var/cache/apk/*
```

Comment:
Thanks.  I'll add it into the next commit for the Dockerfile and Readme.md

somewhat related...once all this is submitted, i'll file an issue to begin packaging this as an alpine package directly. 
eg. the following uses my own repo
(note, its _untrusted_ and i'll probably delete the repo links below) 
```
apk --no-cache add google-cloud-sdk --repository https://storage.googleapis.com/mineral-minutia-820/alpine/v3.5/community --allow-untrusted
apk --no-cache add google-cloud-sdk=150.0.0-r0 --repository https://storage.googleapis.com/mineral-minutia-820/alpine/v3.5/community --allow-untrusted
apk --no-cache add google-cloud-sdk=151.0.0-r0 --repository https://storage.googleapis.com/mineral-minutia-820/alpine/v3.5/community --allow-untrusted
```